README.md

```diff
@@ -37,20 +37,20 @@ $workers = ($httpResults) ? $httpResults->workers : [];
 ```
 
 ### Methods
-  - #### `getWorkersMeta()`
+  - #### `getWorkersMeta(): HttpResponse`
      + sends a GET request to retrieve workers api meta data.
      
      ```php 
      $adp->getWorkersMeta();
      ```
-  - #### `getWorker(string $aoid, array $select = []);`
+  - #### `getWorker(string $aoid, array $select = []): HttpResponse`
```

Comment:
what's `aoid`?

Reply:
`aoid` stands for `Associate OID`, I imagine `OID` being `Organizational ID`